File: 2-Basic-Mathematical-Knowledge-for-Algorithms/006-Print_2N+3/typescript/main.ts
import * as readline from 'readline';

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

rl.on('line', (line: string) => {
    // 1. 入力処理
    const n: number = parseInt(line.trim(), 10);

    // 2. 計算処理
    const result: number = 2 * n + 3;

    // 3. 出力処理
    console.log(result);
    
    rl.close();
});

File: 2-Basic-Mathematical-Knowledge-for-Algorithms/006-Print_2N+3/rust/src/main.rs
use std::io;

fn main() {
    // 1. 入力処理
    let mut input = String::new();
    io::stdin().read_line(&mut input).expect("Failed to read line");
    let n: i32 = input.trim().parse().expect("Please enter a number");

    // 2. 計算処理
    let result = 2 * n + 3;

    // 3. 出力処理
    println!("{}", result);
}